lib/model/ProfielModel.class.php

```diff
@@ -50,11 +50,11 @@ public static function existsUid($uid) {
 	}
 
 	public static function existsNick($nick) {
-		return Database::sqlExists(static::getTableName(), 'nickname = ?', array($nick));

```

Comment:
Gaat dit goed? `existsNick` is static en in de static context is `$this` niet gedefinieerd.